u2-routing-controllers/app.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express')
+const { urlencoded, json } = require('body-parser')
+const { getUserByQuery, getUserByPath, getUserByForm, search } = require('./features/userController')
+const { logger } = require('./_services/logger')
+const { indexC } = require('./features/indexController')
+const { ProductSearch } = require('./features/productController')
+const { RGB } = require('./features/rgbController')
```

Comment:
ฉันแนะนำให้คุณตั้งชื่อตัวแปรทั้งหมดโดยใช้ `lowerCamelCase':
I recommend that you name all variables using `lowerCamelCase`:
- `productSearch`
- `rgb`

Reply:
OK teacher, Should I fix it?

Reply:
ไม่ใช่เรื่องเร่งด่วนหรือสำคัญ แต่ถ้าคุณมีเวลา การแก้ไขจะช่วยให้คุณจำอนุสัญญานี้ได้ :)
It's not urgent or important, but if you have time then fixing it will help you to remember this convention :)